joomla/joomla-cms
# Feature/40545 remove inline padding on highlight search words

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/40545

### Summary of Changes

The issue was caused by merging code changes on highlight.js in PR https://github.com/joomla/joomla-cms/pull/38908 solving 
This PR will revert those changes and solve the initial issue https://github.com/joomla/joomla-cms/issues/38903
Issue solved by reset inline padding on the search words by overriding Bootstrap variable `$mark-padding`

### Testing Instructions

Follow testing instruction as described by both issues #40545 and #38903

After applying the patch the command `npm run build` should be executed to regenerate both js and css files. 

### Actual result BEFORE applying this Pull Request

- Search word `Administrator` will not be highlighted when it is followed by a dot or comma.
- Search word `Foo Bar FooBar` will be highligted

### Expected result AFTER applying this Pull Request

- Search word `Administrator` will be highligted
- Search word `Foo Bar FooBar` will be highligted

### Link to documentations
Please select:
